packages/pwa-buildpack/package.json

```diff
@@ -52,6 +52,7 @@
     "pkg-dir": "~4.1.0",
     "portscanner": "~2.2.0",
     "redis": "~2.8.0",
+    "require-graphql-file": "~1.0.1",
```

Comment:
Needed to be able to `require` `.graphql` files at build time.

Reply:
Seems like it just lets you leave off the `.graphql` which doesn't seem that beneficial.
If we need to use this utility we should probably just [roll our own](https://github.com/Ikos9/require-graphql-file/blob/master/src/index.js) as this was last published 2 years ago.

Reply:
I definitely thought about rolling our own, yeah. It's just reading the contents of the file and returning back a string.

I can do that if you want.

Reply:
We can do it ourselves if all it does is the conversion. We can use `fs` for this.

Reply:
It turns `myFileName` into `myFileName.graphql` and then reads that file from the path. I wonder if `require('myFileName.graphql')` would just work.

Reply:
Unfortunately `require('myFileName.graphql')` doesn't work.

I'm not an expert on `require` but I just need the contents of the file as a string, and I think `require` is trying to turn it into an object or something that can be understood by other code.

Anyway, I removed this dependency and wrote our own 👍